test/unit/api/ddi-index-api/persons.test.js

```diff
@@ -0,0 +1,45 @@
+
+const { getPersons } = require('../../../../app/api/ddi-index-api/persons')
+const { get } = require('../../../../app/api/ddi-index-api/base')
+jest.mock('../../../../app/api/ddi-index-api/base')
+
+describe('Persons test', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe('getPersons', () => {
+    test('should get people filtered by firstName and lastName', async () => {
+      get.mockResolvedValue({ payload: {} })
+      await getPersons({
+        firstName: 'Homer',
+        lastName: 'Simpson'
+      })
+      expect(get).toHaveBeenCalledWith('persons?firstName=Homer&lastName=Simpson', expect.anything())
```

Comment:
Do we need to do anything special, either in terms of param validation or url encoding, when a surname may contain spaces e.g. Van de Wert'

Reply:
<img width="521" alt="Screenshot 2024-03-08 at 12 51 25" src="https://github.com/DEFRA/aphw-ddi-portal/assets/56303993/059c8abb-4403-4c0d-8170-fbc7d3105e86">

Reply:
URLSearchParams prevents does it all for us, and sanitises the query as well, so no sql injection attempts